apps/typegpu-docs/src/examples/image-processing/background-segmentation/schemas.ts

```diff
@@ -15,6 +15,8 @@ export const prepareModelInputLayout = tgpu.bindGroupLayout({
   sampler: { sampler: 'filtering' },
 });
 
+export const cropBoundsSlot = tgpu.slot<TgpuUniform<d.Vec4f>>();
```

Comment:
I'd recommend an accessor for this usecase instead. We don't really care that it's a uniform, rather that it's a vec4f

```suggestion
export const cropBoundsAccess = tgpu['~unstable'].accessor(d.vec4f);
```